_posts/2020-06-16-aws-acronyms-overview.md

```diff
@@ -0,0 +1,153 @@
+
+**CFN (CloudFormation) allows us to define our infrastructure 'stack' in YML files containing a group of resources (think EC2, ELB, Security Groups, RDS instances, etc).** 
+
+We create and update our infrastructure by executing these from AWS console, CLI, or integrating with our CI/CD pipelines. CloudFormation is a widely used service and is integral to all automation and provisioning activities.
+
+## Run Containers On ECS & EKS
+Our container infrastructure requires a registry like Docker hub to publish our images and an orchestration system for running the desired number of container instances across multiple host machines like EC2 instances. AWS provides [ECR](https://aws.amazon.com/ecr) for image registry and [ECS](https://aws.amazon.com/ecs) for container orchestration.  
+
+**ECR (Elastic Container Registry) provides a private registry for publishing our container images with access controlled by IAM policies**.  
+
+**ECS (Elastic Container Service) is the container orchestration service for running stateless and stateful containers using tasks and services**. Here are some hands-on [series of articles](https://reflectoring.io/aws-deploy-docker-image-via-web-console/) for learning about running your applications on ECS.
+
+**EKS (Elastic Kubernetes Service) is the fully-managed Kubernetes offering from AWS.** EKS provides a managed control plane and managed worker nodes.
+
+Both ECS and EKS come with a Fargate option for provisioning EC2 instances. AWS Fargate is a serverless compute for containers and removes the need to provision and manage servers.
+
+## Serverless Compute With Lambda and SAM
+With [AWS lambda](https://aws.amazon.com/lambda) we can eliminate activities of provisioning servers of the right capacity.
+**Lambda is the service for running functions in a serverless model.** We provide our function written in one of the supported languages with enough permissions to execute. 
+The server for executing the function is provisioned at the time of invocation. The infrastructure is dynamically scaled, depending on the number of concurrent requests. 
+**Lambda is commonly invoked by events from other AWS services like API Gateway, SQS, SNS, or Cloudwatch**.
+
+**SAM (Serverless Application Model) is the framework for developing lambda applications with useful tools like a CLI, a local test environment based on docker, and integration with developer tools.** 
+ 
+## Deliver Content With CloudFront
+[AWS CloudFront](https://aws.amazon.com/cloudfront) is a CDN-content delivery network service to serve content from the nearest AWS POP (point of presence). **It is a global autoscaled service integrated for content origin with resource endpoints in EC2, ELB, and S3.** 
```

Comment:
I don't understand the sentence "It is a global autoscaled service integrated for content origin with resource endpoints in EC2, ELB, and S3.". Can you please rephrase that?